ophyd/tests/test_utils.py

```diff
@@ -32,19 +32,6 @@ def test_waveform_to_string():
     assert epics_utils.waveform_to_string(asc) == s
 
 
-def test_pyepics_version_support():
```

Comment:
Kind of an unnecessary test, but did it really need to be blown up? Wouldn't changing `good_versions` to `_min_pyepics` future proof this?

Reply:
This was to test some weird strings we were getting out of pyepics 4 years ago, given that this also pins to pyepics > 3.4.0 we don't need to handle those cases anymore.

Reply:
also, we are now using `distutils` to parse the version strings so I don't think we need to test that they are working correctly.